lib/commander/help_formatters/terminal/help.erb

```diff
@@ -6,10 +9,11 @@
 
     <%= Commander::HelpFormatter.indent 4, program(:description) %>
 
-  <%= $terminal.color "COMMANDS", :bold %>:
+  <%= "%s: %s" % [$terminal.color("COMMANDS", :bold), default_command.nil? ? '' : '(* default)'] %>
```

Comment:
Also this doesn't seem that clear to me... instead of indicating the default with an asterisk, why not simply label the actual default command below with "(default)" ? That seems more obvious.